packages/web/public/favicon/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
```

Comment:
This some of this content needs adding, I'd ask @willcompanion or @mylespalmer for the theme color